### ISSUE TITLE
Undefined: tea.QuitMsg

### ISSUE THREAD

**Issue by AvicennaJr** [aschey/bubbleprompt#1]
I get the following errors when trying some examples:

`go/pkg/mod/github.com/aschey/bubbleprompt@v0.0.0-20230324225504-9d5c4571956f/prompt.go:106:23: undefined: tea.QuitMsg`

**Comment by aschey** [comment]
Unfortunately, this currently requires using a `replace` directive to use my fork of Bubble Tea until this PR: https://github.com/charmbracelet/bubbletea/pull/536 is merged. See here: https://github.com/aschey/bubbleprompt/blob/main/go.mod#L53

Honestly, I probably won't be developing this project any further. Parts of the code are quite messy and I'm starting to work on a different approach to this problem using the language server protocol.

**Comment by AvicennaJr** [comment]
Alright, I'll replace it and give it a try.

And that sucks but I understand, you have to be selective with where you put your effort.

Go-prompt is cool and I think I'll use that, but bubbleprompt would have been way cooler. Once I'm more confident with my go skills I'll perhaps help work on this.

**Comment by aschey** [comment]
Yeah, go-prompt is pretty great. I would probably continue to clean this up instead of abandoning it, but most of my other work is in Rust and I have some other Rust projects that would benefit from something like this, so I'll probably focus my efforts there. If you'd like to pick this up, I'd be happy to help with any questions you have.

**Comment by AvicennaJr** [comment]
Cool cool. I'll comeback here once I'm ready.

And its Rust™ now, don't want to get sued now do you 😂

**Comment by aschey** [comment]
Oh man, don't tell the cops! I'll lose my Rust™ street cred :fearful:
